spec/models/activity_spec.rb

```diff
@@ -77,8 +77,8 @@
       end
     end
 
-    describe ".publishable_to_iati?" do
-      it "only returns activities with a `form_state` of :complete and/or `publish_to_iati` of true" do
+    describe ".publishable_to_iati" do
+      it "only returns activities with a 'complete' and `publish_to_iati` is true" do
```

Comment:
While you're in there making things better (🎉 ), could you rephrase "with a 'complete'" to something else?